morpheus/stages/input/arxiv_source.py

```diff
@@ -34,12 +34,24 @@
 logger = logging.getLogger(__name__)
 
 
+@register_stage("from-arxiv")
 class ArxivSource(SingleOutputSource):
-
-    def __init__(self, c: Config):
+    """
+    Source stage that downloads PDFs from arxiv and converts them to dataframes
+    
+    Parameters
+    ----------
+    c : `morpheus.config.Config`
+        Pipeline configuration instance.
+    query : `str`, default = "large language models"
+        Query to use for arxiv search.
```

Comment:
I dont think there should be a default here. You can use this default in the documentation but it should not be the think we search for automatically